docs/tutorial/tutorial.rst

```diff
@@ -546,6 +547,85 @@ Also note, that :func:`.GridLayout.generate_layout` returns `two` values. We hav
 ``layout_cell``. The value in ``mapping`` will tell you where each device was placed. To make use of this, you have to
 pass a unique id when calling ``add_to_row``.
 
+Generating electron beam lithography markers
+============================================
```

Comment:
Maybe start with explaining how markers can individually be placed using `add_ebl_marker`? The `evl_frame` in just uses this function for a given layout

Reply:
done

Reply:
done